### DISCUSSION TITLE
Download Linter Cop

### DISCUSSION THREAD

**Discussion by Axarath** [StefanMaron/BusinessCentral.LinterCop#642] [Q&A]
Hi,

after upgrade to AL 13.0.1027618 Linter Cop dll is not available and can't be downloaded. I can copy it manually but it is a bit annoying.
Did you face the same issue ?

**Comment by Arthurvdv** [comment]
I see that I recompiled the release version, but not the pre-release version of the LinterCop for this new version of AL (sorry!).

It should work again now I think.
